axiom/nr_php_packages.h

```diff
@@ -43,7 +49,10 @@ typedef struct _nr_php_packages_t {
  *           nr_php_packages_add_package() is not called, then it must be freed
  *           by nr_php_package_destroy()
  */
-extern nr_php_package_t* nr_php_package_create_with_source(char* name, char* version, const nr_php_package_source_priority_t source_priority);
+extern nr_php_package_t* nr_php_package_create_with_source(
+    const char* name,
+    const char* version,
+    const nr_php_package_source_priority_t source_priority);
```

Comment:
Formatting only, right?

Reply:
Yes I ran it through `git-clang-format`.

Reply:
No I changed the pointers to be `const` as I saw no reason for them to not be.
Then `git-clang-format` did the rest.